src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -54,7 +54,7 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
         <FastImage source={picture} style={[styles.picture, styles.anonBgColor]} />
       )}
       <Text numberOfLines={1} ellipsizeMode="tail" style={styles.name}>
-        as {name}
+        {type === ANONYMOUS ? name : `as ${name}`}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ternary operator here changes the display name based on the `type`. This could potentially lead to confusion if the `name` variable does not clearly indicate whether the user is anonymous or not. Consider adding more explicit labeling for clarity.
```diff
- {type === ANONYMOUS ? name : `as ${name}`}
+ {type === ANONYMOUS ? `Anonymous: ${name}` : `Signed: ${name}`}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->